.github/workflows/build-docker.yml

```diff
@@ -11,9 +11,15 @@ jobs:
       GROCY_IMAGE_TAG: ${{ github.event.release.tag_name }}
     steps:
     - uses: actions/checkout@v2
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
```

Comment:
Could we add a comment of some kind to indicate that this is a dependency for `bake-action`?

Reply:
Yeah for sure.

Reply:
Thanks for adding that.  It probably seems trivial, and hopefully there'll be tooling in future to identify non-required dependencies.. but until then, having a comment serves to indicate why each setup action exsists.